### ISSUE TITLE
6.1.0 causes CI to fail

### ISSUE THREAD

**Issue by kellyselden** [aexmachina/ember-cli-sass#151]
"Error: File to import not found or unreadable: util/util."
https://travis-ci.org/kellyselden/ember-tree/jobs/190191206#L565
https://github.com/kellyselden/ember-tree/issues/143

**Comment by aexmachina** [comment]
Thanks @kellyselden, where is the `util/util` import coming from?

**Comment by kellyselden** [comment]
Not sure, here is the full stack trace from CI:
```
[31mFile: /home/travis/build/kellyselden/ember-tree/tmp/sass_compiler-input_base_path-vWojca63.tmp/app/styles/_settings.scss (44)[39m
[31mThe Broccoli Plugin: [SassCompiler] failed with:[39m
Error: File to import not found or unreadable: util/util.
Parent style sheet: /home/travis/build/kellyselden/ember-tree/tmp/sass_compiler-input_base_path-vWojca63.tmp/app/styles/_settings.scss
    at options.error (/home/travis/build/kellyselden/ember-tree/node_modules/ember-cli-sass/node_modules/broccoli-sass-source-maps/node_modules/node-sass/lib/index.js:286:26)

The broccoli plugin was instantiated at: 
    at SassCompiler.Plugin (/home/travis/build/kellyselden/ember-tree/node_modules/ember-cli-sass/node_modules/broccoli-sass-source-maps/node_modules/broccoli-caching-writer/node_modules/broccoli-plugin/index.js:7:31)
    at SassCompiler.CachingWriter (/home/travis/build/kellyselden/ember-tree/node_modules/ember-cli-sass/node_modules/broccoli-sass-source-maps/node_modules/broccoli-caching-writer/index.js:18:10)
    at new SassCompiler (/home/travis/build/kellyselden/ember-tree/node_modules/ember-cli-sass/node_modules/broccoli-sass-source-maps/index.js:20:17)
    at /home/travis/build/kellyselden/ember-tree/node_modules/ember-cli-sass/index.js:38:12
    at Array.map (native)
    at SASSPlugin.toTree (/home/travis/build/kellyselden/ember-tree/node_modules/ember-cli-sass/index.js:35:34)
    at /home/travis/build/kellyselden/ember-tree/node_modules/ember-cli/node_modules/ember-cli-preprocess-registry/preprocessors.js:180:26
    at Array.forEach (native)
    at processPlugins (/home/travis/build/kellyselden/ember-tree/node_modules/ember-cli/node_modules/ember-cli-preprocess-registry/preprocessors.js:178:11)
    at module.exports.preprocessCss (/home/travis/build/kellyselden/ember-tree/node_modules/ember-cli/node_modules/ember-cli-preprocess-registry/preprocessors.js:148:10)
    at EmberAddon.EmberApp.styles (/home/travis/build/kellyselden/ember-tree/node_modules/ember-cli/lib/broccoli/ember-app.js:1319:28)
    at EmberAddon.EmberApp.toArray (/home/travis/build/kellyselden/ember-tree/node_modules/ember-cli/lib/broccoli/ember-app.js:1647:10)
    at EmberAddon.EmberApp.toTree (/home/travis/build/kellyselden/ember-tree/node_modules/ember-cli/lib/broccoli/ember-app.js:1668:30)
    at module.exports (/home/travis/build/kellyselden/ember-tree/ember-cli-build.js:17:14)
    at CoreObject.module.exports.Task.extend.setupBroccoliBuilder (/home/travis/build/kellyselden/ember-tree/node_modules/ember-cli/lib/models/builder.js:72:19)
    at CoreObject.module.exports.Task.extend.init (/home/travis/build/kellyselden/ember-tree/node_modules/ember-cli/lib/models/builder.js:52:10)
```

**Comment by zen-tmq** [comment]
I am running into this issue as well. 

```
ember-cli: 2.10.0
node: 4.7.0
os: darwin x64
```

repo steps:
```
ember new consuming-app
ember install ember-cli-sass
ember build
```
```
Build failed.
File: /Users/dhoyer/Documents/scratch/consuming-app/tmp/sass_compiler-input_base_path-a0lzbemS.tmp/app/styles/_settings.scss (44)
The Broccoli Plugin: [SassCompiler] failed with:
Error: File to import not found or unreadable: util/util.
Parent style sheet: /Users/dhoyer/Documents/scratch/consuming-app/tmp/sass_compiler-input_base_path-a0lzbemS.tmp/app/styles/_settings.scss
    at options.error (/Users/dhoyer/Documents/scratch/consuming-app/node_modules/node-sass/lib/index.js:286:26)

The broccoli plugin was instantiated at: 
    at SassCompiler.Plugin (/Users/dhoyer/Documents/scratch/consuming-app/node_modules/broccoli-plugin/index.js:7:31)
    at SassCompiler.CachingWriter [as constructor] (/Users/dhoyer/Documents/scratch/consuming-app/node_modules/broccoli-caching-writer/index.js:18:10)
    at new SassCompiler (/Users/dhoyer/Documents/scratch/consuming-app/node_modules/broccoli-sass-source-maps/index.js:20:17)
    at /Users/dhoyer/Documents/scratch/consuming-app/node_modules/ember-cli-sass/index.js:38:12
    at Array.map (native)
    at SASSPlugin.toTree (/Users/dhoyer/Documents/scratch/consuming-app/node_modules/ember-cli-sass/index.js:35:34)
    at /Users/dhoyer/Documents/scratch/consuming-app/node_modules/ember-cli-preprocess-registry/preprocessors.js:180:26
    at Array.forEach (native)
    at processPlugins (/Users/dhoyer/Documents/scratch/consuming-app/node_modules/ember-cli-preprocess-registry/preprocessors.js:178:11)
    at module.exports.preprocessCss (/Users/dhoyer/Documents/scratch/consuming-app/node_modules/ember-cli-preprocess-registry/preprocessors.js:148:10)
```

**Comment by zen-tmq** [comment]
The issue is present in 6.1.0 but not in 6.0.0

**Comment by GCheung55** [comment]
I think this is due to the change `ember-cli-foundation-6-sass` to support `foundation-sites@6.3.0`.

I left a comment about it: https://github.com/acoustep/ember-cli-foundation-6-sass/issues/48#issuecomment-272723908

**Comment by aexmachina** [comment]
This comment should fix your problem: https://github.com/acoustep/ember-cli-foundation-6-sass/issues/48#issuecomment-272304023

**Comment by kellyselden** [comment]
@aexmachina I'm not using any `@import` statements in my sass file. Here are the tiny sass files:

https://github.com/kellyselden/ember-tree/blob/master/addon/styles/addon.scss
https://github.com/kellyselden/ember-tree/blob/master/app/styles/app.scss

**Comment by GCheung55** [comment]
@aexmachina, I cloned a copy of ember-tree. The installation of `ember-cli-sass@6.1.0` still has `app/styles/app.scss` and `app/styles/_settings.scss`.

**Comment by aexmachina** [comment]
Thanks everyone. An `app.scss` was auto-created when I updated `foundation-sites`. I didn't `git add` it, but it was shipped to npm because it was on the filesystem. Fixed in 6.1.1.

**Comment by kellyselden** [comment]
Working for me now. Thanks @aexmachina and everyone!
